applications/PfemFluidDynamicsApplication/custom_elements/two_step_updated_lagrangian_V_P_implicit_element.cpp

```diff
@@ -19,374 +19,381 @@
+    }
+    }
+
+    KRATOS_CATCH("");
   }
 
-  default:
+  template <unsigned int TDim>
+  void TwoStepUpdatedLagrangianVPImplicitElement<TDim>::CalculateLocalMomentumEquations(MatrixType &rLeftHandSideMatrix, VectorType &rRightHandSideVector, ProcessInfo &rCurrentProcessInfo)
   {
-    KRATOS_THROW_ERROR(std::logic_error, "Unexpected value for TWO_STEP_UPDATED_LAGRANGIAN_V_P_ELEMENT index: ", rCurrentProcessInfo[FRACTIONAL_STEP]);
-  }
-  }
+    KRATOS_TRY;
 
-  KRATOS_CATCH("");
-}
+    GeometryType &rGeom = this->GetGeometry();
+    const unsigned int NumNodes = rGeom.PointsNumber();
+    const unsigned int LocalSize = TDim * NumNodes;
 
-template <unsigned int TDim>
-void TwoStepUpdatedLagrangianVPImplicitElement<TDim>::CalculateLocalMomentumEquations(MatrixType &rLeftHandSideMatrix, VectorType &rRightHandSideVector, ProcessInfo &rCurrentProcessInfo)
-{
-  KRATOS_TRY;
+    MatrixType MassMatrix = ZeroMatrix(LocalSize, LocalSize);
+    MatrixType StiffnessMatrix = ZeroMatrix(LocalSize, LocalSize);
 
-  GeometryType &rGeom = this->GetGeometry();
-  const unsigned int NumNodes = rGeom.PointsNumber();
-  const unsigned int LocalSize = TDim * NumNodes;
+    // Check sizes and initialize
+    if (rLeftHandSideMatrix.size1() != LocalSize)
+      rLeftHandSideMatrix.resize(LocalSize, LocalSize, false);
 
-  MatrixType MassMatrix = ZeroMatrix(LocalSize, LocalSize);
-  MatrixType StiffnessMatrix = ZeroMatrix(LocalSize, LocalSize);
+    rLeftHandSideMatrix = ZeroMatrix(LocalSize, LocalSize);
 
-  // Check sizes and initialize
-  if (rLeftHandSideMatrix.size1() != LocalSize)
-    rLeftHandSideMatrix.resize(LocalSize, LocalSize, false);
+    if (rRightHandSideVector.size() != LocalSize)
+      rRightHandSideVector.resize(LocalSize);
 
-  rLeftHandSideMatrix = ZeroMatrix(LocalSize, LocalSize);
+    rRightHandSideVector = ZeroVector(LocalSize);
 
-  if (rRightHandSideVector.size() != LocalSize)
-    rRightHandSideVector.resize(LocalSize);
+    // Shape functions and integration points
+    ShapeFunctionDerivativesArrayType DN_DX;
+    Matrix NContainer;
+    VectorType GaussWeights;
+    this->CalculateGeometryData(DN_DX, NContainer, GaussWeights);
+    const unsigned int NumGauss = GaussWeights.size();
+    const double TimeStep = rCurrentProcessInfo[DELTA_TIME];
 
-  rRightHandSideVector = ZeroVector(LocalSize);
+    double theta = this->GetThetaMomentum();
 
-  // Shape functions and integration points
-  ShapeFunctionDerivativesArrayType DN_DX;
-  Matrix NContainer;
-  VectorType GaussWeights;
-  this->CalculateGeometryData(DN_DX, NContainer, GaussWeights);
-  const unsigned int NumGauss = GaussWeights.size();
-  const double TimeStep = rCurrentProcessInfo[DELTA_TIME];
+    ElementalVariables rElementalVariables;
+    this->InitializeElementalVariables(rElementalVariables);
 
-  double theta = this->GetThetaMomentum();
+    double totalVolume = 0;
+    double MeanValueMass = 0;
```

Comment:
should be cammel_case like mean_value_mass but I think that the whole file should be refactored eventually